### ISSUE TITLE
why using bce as loss function

### ISSUE THREAD

**Issue by zxu47** [justHungryMan/vision-transformer-tf#1]
Hi

     I am using your code to train ViT from scratch. I do not know why you use BCE as the loss function of the upstream job and CE as the loss function of the downstream task. Is this the original design of the ViT?

**Comment by justHungryMan** [comment]
Hi @zxu47 , thank you for your interest.

Original ViT also trained with BCE on pretrain and CE on finetuning.

You can check [here](https://github.com/google-research/vision_transformer/issues/153) 

And also, we provide [coyo-vit](https://github.com/kakaobrain/coyo-vit) pretrain & finetuning code (which used coyo-labeled-300m dataset) , weights and results.
